spec/controllers/gradings_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe Assessment::GradingsController do
+  let(:lecturer) {FactoryGirl.create(:lecturer)}
+  let!(:course) { FactoryGirl.create(:course, creator: lecturer) }
+  let(:mission) { FactoryGirl.create(:mission, :with_general_questions, :with_coding_questions, :completed, course: course) }
+
+  describe 'GET new' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.